src/main/java/com/matuageorge/technicalmaintenanceschedule/TechnicalMaintenanceScheduleApplication.java

```diff
@@ -10,4 +12,9 @@ public static void main(String[] args) {
         SpringApplication.run(TechnicalMaintenanceScheduleApplication.class, args);
     }
 
+    @Bean
+    ModelMapper modelMapper() {
+        return new ModelMapper();
+    }
+
```

Comment:
please create a separate config class for this bean. Don't put configurations in SpringBootApplication class.